index.html

```diff
@@ -61,6 +61,95 @@
                 </div>
             </header>
         </div>
+
+        <!-- Hero section -->
+        <section id="#" class="container hero-container">
+            <div class="hero">
+                <!-- Gradient Div -->
+                <div class="hero__gradient"></div>
+                <div class="hero__main">
+                    <div class="hero__main-image">
+                        <img src="/assets/images/hero.svg" alt="hero-image" />
+                    </div>
+                    <div class="hero__main-content">
+                        <div class="hero__content-texts">
+                            <button
+                                class="button button__filled--light-pink hero_content-buttons--pink"
+                            >
+                                Explore the world!
+                                <span class="icon-suitcase"></span>
+                            </button>
+                            <h1 class="heading1 hero__content-header">
+                                Travel <span> top destination</span> of the
+                                world
+                            </h1>
+                            <p class="body1 hero__content-body">
+                                We always make our customer happy by providing
+                                as many choices as possible
+                            </p>
+                        </div>
+                        <div class="hero__content-buttons">
+                            <button
+                                class="button button__filled--primary hero__buttons--primary"
+                            >
+                                Get Started
+                            </button>
+                            <button
+                                class="button button__outline--primary hero__bu white-space: pre-wrap;ttons--outline"
```

Comment:
```suggestion
                                class="button button__outline--primary hero__buttons--outline"
```

why do we need two modifiers here?

Reply:
To give a unique identity for future scope , if needed some action on it.

Reply:
are we using the other class for any styling purposes here?

Reply:
yes